### DISCUSSION TITLE
Accessibility v4 versus v5

### DISCUSSION THREAD

**Discussion by bob-dlc** [twbs/bootstrap#38250] [Q&A]
We're using Bootstrap 4.4 on a project and are wondering whether upgrading to Bootstrap 5.3 would provide us additional features to  improve the accessibility of our app. Unfortunately I can't find any change log information across the versions. Does anyone know how accessibility support has been evolving in Bootstrap?

**Comment by julien-deramond** [comment]
Precisely around accessibility, it's a bit difficult to answer since our components are always evolving.
You may have information in our [Migrating to v5 guide](https://getbootstrap.com/docs/5.3/migration/).
IDK if it can be helpful but you might have more precise info by filtering the PRs in v5: https://github.com/twbs/bootstrap/pulls?q=is%3Apr+is%3Aclosed+label%3Av5+label%3Aaccessibility. Maybe some accessibility labels are missing but it could give you an idea on the work done specifically in this field.
